test/statement_types.py

```diff
@@ -130,7 +130,10 @@ def __init__(self, overload, name, stan_arg, size, value=None):
         self.size = size
 
         if value is None:
-            self.value = 0.4
+            if 'complex' in stan_arg:
```

Comment:
[optional]
I prefer ternary operators when the body of conditionals only assign to the same variable.  But ternary ops look pretty awkward in Python (of course this one wouldn't even be legal in C++ with its heterogeneous return types), so I'm marking as optional.

```python
self.value = to_complex(0.4, 04) if 'complex' in stan_arg else 0.4
```

Reply:
I agree in general but quite dislike the Python ternary operator's ordering (I'm much more used to the order `conditional - true value - false value` from C, Java, etc., so I still find `true value - conditional - false value` hard to parse even after using it for years)

Reply:
Glad to hear it's not just me finding it hard to read.  That is why I marked it optional.

Reply:
+1 on the Python ternary being hard to read :)

Reply:
I think it was all to avoid making something like `then` a keyword. OCaml's if expression keeps the same order as C's ?: by having the style be `if cond then true_value else false_value`, which is probably my favorite option